### ISSUE TITLE
Read IO stats from iostat if /proc/diskstats is empty

### ISSUE THREAD

**Issue by thijsc** [appsignal/probes-rs#35]
In LXC containers it seems like `/proc/diskstats` is empty, while `iostats` does give relevant information. `iostats` is not available everywhere. Maybe fall back on it if `/proc/diskstats` is empty?

**Comment by tombruijn** [comment]
We should probably search this in the `/sys/fs/` directory instead.

**Comment by vbmade2000** [comment]
is **iostats** external utility ? I checked in my Ubuntu 20 but didn't find it.

**Comment by tombruijn** [comment]
@vbmade2000 yes I think it's an external utility and it's not installed by default.
